alpine-glibc/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM arm64v8/alpine:3.9
```

Comment:
Unrelated to ticket crosscloudci/crosscloudci#103.  This is used to build images manually.  Normally happens on master.